homeassistant/components/hassio/__init__.py

```diff
@@ -124,6 +126,33 @@
 }
 
 
+async def async_addon_is_installed(
+    hass: HomeAssistantType, slug: str, repository: str
+) -> bool:
+    """Return True if add-on is installed.
+
+    The caller of the function should handle HassioAPIError.
+    """
+    command = "/supervisor/info"  # only installed addons are returned
+    result = await async_send_supervisor_command(hass, command, method="get")
+    addons = result["data"]["addons"]
+    found_addon = next(
+        (_addon for _addon in addons if _addon["slug"] == f"{repository}_{slug}"), None,
+    )
+    return bool(found_addon)
+
+
+async def async_send_supervisor_command(
```

Comment:
I don't think that we should expose this directly because it means the integration no longer has control over what is and is not used.

Reply:
Ok. Maybe just inline it inside each public helper? Or make it a private helper.